src/visualizer/sortingvisualizer/util/GeneralUtil.js

```diff
@@ -85,16 +135,69 @@ const generateDefaultArray = (size) => {
   return array;
 };
 
-// Conditionals
+/**
+ * Returns true if the algorithm input is counting sort.
+ *
+ * @param {string} visualizerAlgorithm Input algorithm.
+ * @returns {boolean} True iff the algorithm is counting sort.
```

Comment:
Was wondering `iff` is typo since the `if` in line 139 is `if`,

Reply:
Noted, will fix.